src/solvers/dgsem_tree/subcell_limiters_2d.jl

```diff
@@ -13,7 +13,12 @@ function create_cache(limiter::Type{SubcellLimiterIDP}, equations::AbstractEquat
                                                                         nnodes(basis),
                                                                         bound_keys)
 
-    return (; subcell_limiter_coefficients)
+    idp_bounds_delta = Dict{Symbol, real(basis)}()
```

Comment:
Since this is effectively where the data structure `idp_bounds_delta` is created, I suggest to add a small comment that outlines its purpose